index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section class="Main"> <!-- SECTION ------------------------------------------>
+        <header class="Main__header">
+            <div class="Main__header_box">
+                <img class="Main__header_box_logo" src="images/logo.svg" alt="logo NOVA">
+            </div>
+            <ul class="Main__header__menu">
+                <li class="Main__header__menu__item">
+                    <a class="Main__header__menu__item__link" href="#">The team</a>
+                </li>
+                <li class="Main__header__menu__item">
```

Comment:
Jeśli to BEM to nie możemy w nieskończoność robić `... _ ... _ ... _` mamy tylko 2 zagłębienia tj. `BLOCK__ELEMENT--MODIFIER` - koniec ;P